user_guide/intro_nav_shortcuts.rst

```diff
@@ -0,0 +1,49 @@
+.. _user-guide-getting-started-shortcuts:
+
+Shortcuts
+=========
+
+Shortcuts are used to perform common actions quickly from anywhere. They can be, particularly, used to create new 
+records and navigate through the system quickly. Shortcuts are at hand, if you are not sure where in the menu the entity 
+you’re looking for is.
+
+Shortcuts are accessible via the icon in the navigation panel.
+
+|
+
+.. image:: ./img/navigation/panel/shortcut_full.png
+
+|
+
+From here you can:
+
+* Launch an action from the "most used" list
+
+.. hint::
+
+    This list is updated as you are using the system, and will initially contain the actions that you use the most.
+
+* Start typing the name of a related entity or an action to choose from a list of matching items:
+
+  |shortcut|
+  
+|
+
+- To see all the shortcut actions available, click *"See the full list"* link
```

Comment:
Edit: To see all the available shortcut actions, click *"See the full list"* link